pages/students.tsx

```diff
@@ -66,20 +66,49 @@ There are three stages for both designers and developers. We start with a resume
 
 Interested in applying? [Click here!](${recruitment.applicationUrl})
 
+### What computer science knowledge is expected for the developer recruitment process?
+
+Here is the knowledge we expect for each grade level. The interview questions might not cover all of these topics, and it might be possible to pass the interviews without knowing all of these topics, but our interview questions are designed with these expectations in mind.
+
+#### Freshmen
+
+- Familiarity with at least one programming language
+- Basic datatypes: booleans, integers, floating-point numbers, strings, arrays/lists
+- Control flow structures: if/else statements, for/while loops
+- Functions
+- Exceptions
+- Unit testing concepts (no specific frameworks needed) and edge cases
+
+#### Sophomores
+
+- All of the topics for freshmen, listed above
+- Pointers or references, depending on the language you use
+- Recursion
+- Data structures: multidimensional arrays/lists, linked lists, hash tables (also known as hash maps, dictionaries)
+- Algorithms: sorting, binary search, greedy
+- Object-oriented programming concepts: classes, constructors, instance variables, methods
+
+#### Juniors/Seniors
+
+- All of the topics for freshmen and sophomores, listed above
+- Designing JSON-compatible data representations in the language of your choice (don't worry about serialization/deserialization though)
+- Analyzing time and memory complexity
+- Data structures: stacks, queues, heaps, trees, and graphs
+- Algorithms: divide and conquer, recursion and backtracking, memoization, _basic_ dynamic programming
+
 ### How can I prepare for the technical components of the developer recruitment process?
 
-To prep for the systems design question:
```

Comment:
why did we remove this? don't we want to share this info?

Reply:
we no longer have a systems design question in the phone screen!